src/utils/TreasuryRevenue.ts

```diff
@@ -98,6 +100,25 @@ export function updateTreasuryRevenueClaimUsdplusReward(block: BigInt, claim: Cl
   treasuryRevenue.save()
 }
 
+export function updateTreasuryRevenueClaimUsdcReward(block: BigInt, claim: ClaimReward): void {
```

Comment:
We should separate MAI and USDC fee in UniV3 strategy. Using `ClaimToken` event should be easier.

Reply:
Ah okay, latest commit adds Mai and Usdc to Revenue. 

What about the impermanent loss?  I'd rather use the "realised" profit for the APR calculation in the Investments page, so should I use PayoutReward there?

Reply:
Yes, `PayoutReward` is staker actually get and `ClaimToken` is revenue from the strategy.